src/nwb_benchmarks/benchmarks/time_remote_slicing.py

```diff
@@ -20,9 +20,9 @@
     EcephysTestCase=dict(
         s3_url=get_s3_url(
             dandiset_id="000717",
-            dandi_path="sub-IBL-ecephys/sub-IBL-ecephys_ses-3e7ae7c0_desc-18000000-frames-13653-by-384-chunking.nwb",
+            dandi_path="sub-npI3_ses-20190421_behavior+ecephys/sub-npI3_ses-20190421_behavior+ecephys.nwb",
         ),
-        object_name="ElectricalSeriesAp",
+        object_name="ElectricalSeries",  # TODO
```

Comment:
Is this still a TODO item or can the comment be removed?

Reply:
Looks like this was the correct object name, so I removed the TODO